src/Compass.h

```diff
@@ -0,0 +1,53 @@
+#ifndef __COMPASS_H__
+#define __COMPASS_H__
+
+#include "BaseIMU.h"
+
+// I²C-address device
+constexpr uint8_t LIS3MDL_SLAVE_ADDRESS = 0x1C;
+constexpr uint8_t LIS3MDL_SLAVE_ADDRESS_ALT = 0x1E;
+
+// Registers addrress
+constexpr uint8_t LIS3MDL_CTRL_REG2_FS0 = 0x20;
+constexpr uint8_t LIS3MDL_CTRL_REG2_FS1 = 0x40;
+
+// Registers value bits
+constexpr uint8_t LIS3MDL_CTRL_REG3_MD0 = 0x00;
+constexpr uint8_t LIS3MDL_CTRL_REG3_MD1 = 0x02;
+
+// Constans
```

Comment:
```suggestion
// Constants
```
"Constants" is too general comment. Please make it more informative.

Reply:
Done